src/parallel_som.jl

```diff
@@ -110,47 +106,46 @@ function trainSOM_parallel(som::Som, train::Any, len;
 
     dm = distMatrix(som.grid, som.toroidal)
 
-    # make Δs for linear decay of r:
-    # r = Float64(r)
-    dTrain = distribute(train)
-
     codes = som.codes
     global_sum_numerator = zeros(Float64, size(codes))
     global_sum_denominator = zeros(Float64, size(codes)[1])
 
     # linear decay function
     if rDecay
-     if r < 1.5
-         Δr = 0.0
-     else
-         println("in r decay")
-         Δr = (r-1.0) / epochs
-     end
+        if r < 1.5
+            Δr = 0.0
+        else
+            Δr = (r-1.0) / epochs
+        end
     else
-     Δr = 0.0
+        Δr = 0.0
     end
 
+    nWorkers = nprocs()
```

Comment:
What is the difference with `workers()`?

Reply:
workers actually returns an array of worker id's
nprocs returns the number of workers